docs/get-started/configuration.rst

```diff
@@ -1,6 +1,16 @@
 Configuration
 -------------
 
+roster
+======
+
+Enter the path to the classroom roster. If you enter a bare filename,
+`abc-classroom` will look in the course directory for the file. If your
+roster file is somewhere else, you can enter a full path to the file,
+e.g. `/home/user/directory/filename.csv`. 
+
+Default: `roster: classroom_roster.csv`
+=======
```

Comment:
This is causing the build to fail, I don't think this underline should be here? If it should be it needs to be longer to underline the whole line.